r124/u_inref/jnlprocnum.csh

```diff
@@ -0,0 +1,30 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+
+# Get Shared Memory ID of the INST1 JNLPOOL
+$ydb_dist/mupip ftok -jnlpool mumps.repl |& $grep mumps.repl | $tst_awk '{print $6}' >& shm1.out
```

Comment:
Have a "cd $path_INST1" here just in case this script is called from a different directory (just like line 19 for INST3).